test failure with bluez

The command `make test/bluez` fails with the following error:

```
2024/10/17 17:34:15 INFO ERROR: failed to test package. the test environment has been preserved:
2024/10/17 17:34:15 INFO   workspace dir: /tmp/melange-workspace-3963300446
2024/10/17 17:34:15 INFO   guest dir: /tmp/melange-guest-217126860
2024/10/17 17:34:15 ERRO failed to test package: unable to build guest: unable to generate image: installing apk packages: installing packages: installing bluez (ver:5.78-r0 arch:x86_64): unable to install files for pkg bluez: error creating file etc/bluetooth/input.conf: open /tmp/melange-guest-217126860-main/etc/bluetooth/input.conf: permission denied
make: *** [Makefile:136: test/bluez] Error 1
```

--- a/bluez.yaml
+++ b/bluez.yaml
@@ -2,7 +2,7 @@
 package:
   name: bluez
   version: "5.79"
-  epoch: 1
+  epoch: 2
   description: Tools for the Bluetooth protocol stack
   copyright:
     - license: GPL-2.0-or-later AND BSD-2-Clause AND MIT
@@ -58,6 +58,12 @@ pipeline:
 
       install -Dm644 obexd/src/org.bluez.obex.service \
         "${{targets.destdir}}"/usr/share/dbus-1/services/org.bluez.obex.service
+
+      # Workaround permission issue. Fixed upstream, but pulling in patch requires
+      # running autoreconf which seems to generate issues.
+      # https://github.com/bluez/bluez/commit/b1fd409960001a77cda2a09ecc00147ebd9c3667
+      # Fixes: https://github.com/wolfi-dev/os/issues/31026
+      chmod 0755 "${{targets.destdir}}"/etc/bluetooth
 
   - uses: strip
 

